examples/src/main/java/io/grpc/examples/experimental/CompressionServerInterceptor.java

```diff
@@ -0,0 +1,18 @@
+package io.grpc.examples.experimental;
+
+import io.grpc.Metadata;
+import io.grpc.ServerCall;
+import io.grpc.ServerCall.Listener;
+import io.grpc.ServerCallHandler;
+import io.grpc.ServerInterceptor;
+
+public class CompressionServerInterceptor implements ServerInterceptor {
```

Comment:
Add some javadoc?

Reply:
Moved into CompressingHelloWorldServerAllMethods as the review comment below.